pkg/tasks/alertmanager.go

```diff
@@ -106,6 +106,14 @@ func (t *AlertmanagerTask) create(ctx context.Context) error {
 		return errors.Wrap(err, "creating Alertmanager RBAC proxy metric Secret failed")
 	}
 
+	if t.config.ClusterMonitoringConfiguration.AlertmanagerMainConfig.Secrets != nil {
+		for _, secret := range t.config.ClusterMonitoringConfiguration.AlertmanagerMainConfig.Secrets {
+			if err = t.client.CheckSecretExists(ctx, "openshift-monitoring", secret); err != nil {
```

Comment:
```suggestion
			if _, err := t.client.GetSecret(ctx, "openshift-monitoring", secret); err != nil {
```

Reply:
It might also be worth replacing "openshift-monitoring" with a const `MonitoringNamespace`

Reply:
Done